src/html/js/common.js

```diff
@@ -1494,31 +1473,51 @@ Common.getAppAuthToken = function(cellUrl) {
         headers: {'Accept':'application/json'}
     });
 };
+
 /*
- * Get access token for protected box(es) which is accessible by the App.
- * client_id belongs to a App's cell URL
- * Example: MyBoard is "https://demo.personium.io/app-myboard/"
- *          Calorie Smile is "https://demo.personium.io/hn-app-genki/"
+ * Refresh access token for protected box which is accessible by the App.
+ * cellUrl - user's cell URL
  */
-Common.getProtectedBoxAccessToken = function(appToken, cellUrl) {
-  return $.ajax({
-                type: "POST",
-                url: cellUrl + '__token',
-                processData: true,
-                dataType: 'json',
-                data: {
-                    grant_type: "refresh_token",
-                    refresh_token: Common.accessData.refToken,
-                    client_id: Common.getAppCellUrl(),
-                    client_secret: appToken
-                },
-                headers: {
-                    'Accept':'application/json',
-                    'content-type': 'application/x-www-form-urlencoded'
-                }
-            });
+Common.refreshProtectedBoxAccessToken = function(cellUrl) {
+    let engineEndPoint = getRefreshTokenEngineEndPoint();
+    return $.ajax({
+        type: "POST",
+        url: engineEndPoint,
+        processData: true,
+        dataType: 'json',
+        data: {
+            p_target: cellUrl,
+            refresh_token: Common.getRefressToken()

```

Comment:
```suggestion
            refresh_token: Common.getRefreshToken()
```

Reply:
I fixed all the spelling mistakes for getRefreshToken.